src_erl/Communication_Layer/http_Nerlserver/src/MainServer/mainGenserver.erl

```diff
@@ -209,9 +209,9 @@ handle_cast({sourceDone,Body}, State = #main_genserver_state{myName = MyName,sou
 
   case NewCastingList of
     [] -> NextState = State#main_genserver_state{state = idle, sourcesCastingList = NewCastingList,clientsWaitingList = ListOfClients,msgCounter = MsgCounter+1},
-          [{setClientState(clientIdle,ClientName, MyName)}|| ClientName<- ListOfClients],
+          [{setClientState(clientIdle,ClientName, MyName)}|| ClientName<- ListOfClients];
           % gen_server:cast(self(),{clientsIdle}),
-          ack();
+          % ack();
```

Comment:
@halfway258 
Is it intentional the remove of ack? 
If ack doesn't supposed to be here, you can remove it.

Reply:
removed unnecessary ack